sweepbatcher/sweep_batcher.go

```diff
@@ -1085,12 +1109,46 @@ func NewSweepFetcherFromSwapStore(swapStore LoopOutFetcher,
 	}, nil
 }
 
-// fetchSweep fetches the sweep related information from the database.
-func (b *Batcher) fetchSweep(ctx context.Context,
-	sweepReq SweepRequest) (*sweep, error) {
+// fetchSweeps fetches the sweep related information from the database.
+func (b *Batcher) fetchSweeps(ctx context.Context,
+	sweepReq SweepRequest) ([]*sweep, error) {
+
+	if len(sweepReq.Group) != 0 {
+		// Make sure sweepReq.Outpoint, sweepReq.Value were not filled.
+		if sweepReq.Outpoint != (wire.OutPoint{}) {
```

Comment:
... So then we wouldn't need these checks either.

Reply:
Fixed.